### PR TITLE
Code comment for #1591

### PR DESCRIPTION
**Pull request by mkurc-ant**
This is a code comment as requested in #1591

#### Types of changes

- [ ] Bug fix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

#### Checklist:

- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed

### CODE REVIEW

File: vpr/src/pack/prepack.cpp [hunk 1]
```diff
@@ -1548,7 +1548,16 @@ static void update_chain_root_pins(t_pack_patterns* chain_pattern,
     for (const auto pin_ptr : chain_input_pins) {
         std::vector<t_pb_graph_pin*> connected_primitive_pins;
         get_all_connected_primitive_pins(pin_ptr, connected_primitive_pins);
+
+        /**
+         * It is required that the chain pins areconnected inside a complex
+         * block. Although it is allowed to have them disconnected in some of
```
Comment (vaughnbetz) [inline vpr/src/pack/prepack.cpp:1554]: Looks good, thanks.
A couple of typos to fix:
areconnected (missing space)
"in some of modes of the block" -> "in some modes of the block"

**Review by vaughnbetz — APPROVED** [review]
LGTM; just flagged a couple of minor typos.

File: vpr/src/pack/prepack.cpp (hunk 1 above)
Comment (mkurc-ant) [reply vpr/src/pack/prepack.cpp]: Ok, fixed that.

**Comment by vaughnbetz** [comment]
Thanks!

**Pull request merged**